### PR TITLE
pull request after demo

### PR DESCRIPTION
**Pull request by yourgirlcodes**

### CODE REVIEW

File: entities/Event/EventMapping.ts [hunk 1]
```diff
@@ -2,5 +2,5 @@ import {MappingBase} from "../common/mapping";
 import {SchemaId} from "../Schema";
 
 export interface EventMapping extends MappingBase {
-    target: SchemaId;
-}
\ No newline at end of file
+    target?: SchemaId;
```
Comment (iBaryo) [inline entities/Event/EventMapping.ts]: why?.. event mapping always require a target schema

File: entities/common/mapping.ts [hunk 2]
```diff
@@ -1,7 +1,8 @@
 import {Condition, FieldCondition} from "./Condition";
 
 export interface MappingBase {
-    srcField: string;
+    sourceField: string;
     targetField: string;
+    srcField?: string;
```
Comment (iBaryo) [inline entities/common/mapping.ts]: lets drop this field as it's deprecated

File: CDPEntitiesApi.ts [hunk 3]
```diff
@@ -42,11 +35,11 @@ export type CDPEntitiesApi = {
     workspaces: EntityApi<CDPEntityDef<Workspace>, {
         applibrary: EntityApi<CDPEntityDef<Connector>>,
         global: EntityApi<never, {
-            applibrary: EntityApi<CDPEntityDef<Connector>>,
+            applibrary: EntityApi<CDPEntityDef<Connector[]>>,
```
Comment (iBaryo) [inline CDPEntitiesApi.ts]: why? the type provided to `CDPEntityDev` represents the response of a single GET

File: entities/Audience/AudienceCondition.ts [hunk 4]
```diff
@@ -11,83 +11,83 @@ export type AudienceCondition = Condition<
...
-                operand: {
-                    type: 'double',
-                    value: 1000
-                }
-            }
-        }
-    ]
-};
\ No newline at end of file
+// const conditionExample: AudienceCondition = {
```
Comment (iBaryo) [inline entities/Audience/AudienceCondition.ts]: please uncomment this - if something's not compiling then it needs to be fixed.

**Review by iBaryo — CHANGES_REQUESTED** [review]

**Comment by yourgirlcodes** [comment]
relevant for CR

**Comment by yourgirlcodes** [comment]
irrelevant

**Pull request closed without merging**